subprojects/matrix-benchmarking-workloads/rhods-notebooks-ux/plotting/prom.py

```diff
@@ -175,14 +177,22 @@ def get_legend_name(metric_name, metric_metric):
         return name, group
 
     def only_control_plane_nodes(entry, metrics):
-        control_plane_nodes = [node.name for node in entry.results.rhods_cluster_info.control_plane]
+        if entry.is_lts:
+            control_plane_nodes = [node['name'] for node in entry.metadata['cluster_info']['control_plane']]
+        else:
+            control_plane_nodes = [node.name for node in entry.results.rhods_cluster_info.control_plane]
```

Comment:
🤔 
can't you turn the LTS objects into `types.SimpleNamespace()` objects, so that we can remove the `if` here?

Reply:
Even if I change everything in a LTS object to a SimpleNamespace, the structure is different enough to still require an `if`.  I could move this logic to `utils.py` for portability

Reply:
for readability, I would prefer having `types.SimpleNamespace` objects

for `entry.metadata['cluster_info']` vs `entry.results.rhods_cluster_info`, why is `metadata` necessary here? 
I suggested (a few seconds ago) in the MatrixBenchmarking to move it to `entry.results`, which would give:
```
entry.results.metadata.cluster_info
```
but then why not having `entry.results = results["data"] | results["metatata"]` around [this location](https://github.com/openshift-psap/matrix-benchmarking/pull/70/files#diff-c30e98a0916bb9a4504fcbdd364de881530fe711d2932f39e053c61d281a3e96R19)?

all of that would make this `if` necessary

Reply:
[this location](https://github.com/openshift-psap/ci-artifacts/pull/697/files#diff-3bd2f31418733293322aad362b4057396c4e7ff5e384bdd62c5603e68aac53b0R32) actually, not in MatrixBenchmarking